modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js

```diff
@@ -15,15 +15,15 @@ const getCurrentEndDate = (currentEndDate) => {
 
 const ProjectCard = ({code, isSmall, onClick, region, sla, status, title}) => {
 	const getStatusMessage = (status) => {
-		if (status === 1) {
-			return 'Ends on  ';
+		if (status === status.future) {
+			return 'Ends on ';
```

Comment:
@jairmedeiros , @agnosmelo let's remove this extra space soon, it's better to implement this need of an extra space, to function caller.

Reply:
Just to keep it simple.